web/src/test/scala/quasar/api/services/MountServiceSpec.scala

```diff
@@ -80,7 +80,7 @@ class MountServiceSpec extends quasar.QuasarSpecification with ScalaCheck with H
       )
 
       val meff: MEff ~> Task =
-        reflNT[Task] :+: KeyValueStore.fromTaskRef(configsRef)
+        reflNT[Task] :+: KeyValueStore.impl.fromTaskRef(configsRef)
```

Comment:
Ah, I guess I had it in this branch... And I had not yet fixed that compile error. I also fixed it I guess.